user_profile.py

```diff
@@ -56,12 +56,181 @@
 # Create the SQLAlchemy database object
 db.init_app(user_profile_app)
 
+# global resources fields to serialize the response object
+userInformation_resource_fields = {
```

Comment:
nit: Python variable convention is snake case

```suggestion
user_information_resource_fields = {
```

Be sure to update the places where this is used to